secp256k1-sys/README.md

```diff
@@ -26,6 +26,11 @@ $ ./vendor-libsecp.sh depend <version-code> <rev>
 - Where `<version-code>` is the secp256k1-sys version number underscored: `0_1_2`.
 - Where `<rev>` is the git revision of libsecp256k1 to checkout.
 
+## Note
+
+The build dependency `cc` might require a recent version of the rust compiler.
+If you want to compile it with rust 1.22.0, pin its version in your `Cargo.lock`
+with `cargo update -p cc --precise 1.0.41`.
```

Comment:
@apoelstra should we add this note to the main Readme and to the `rust-bitcoin` Readme?

Reply:
I'd like to expand on it a bit more first, but yes.

Reply:
We should also have an IRC discussion with Matt and others about this, because it represents a change in policy (from "works with 1.22 with no work on the user's part, except if they have other packages with incompatible pinning rules, in which case they're fucked" to "works with 1.22 if the user reads the README and futzes around with their lockfiles and generally is conscientious").

Reply:
FWIW, I'm fine with this. It's not perfect but it's a good middle ground. 
By the way, the MSRV RFC  has been merged https://github.com/rust-lang/rfcs/blob/master/text/2495-min-rust-version.md .